types/tests/scales/options.ts

```diff
@@ -17,6 +17,9 @@ const chart = new Chart('test', {
         type: 'time',
         time: {
           unit: 'year'
+        },
+        ticks: {
+          stepSzie: 1
```

Comment:
stepSzie vs stepSize?

Reply:
it was already fixed by https://github.com/chartjs/Chart.js/pull/10962